parboiled-java/src/main/java/org/parboiled/transform/GroupClassGenerator.java

```diff
@@ -72,7 +70,7 @@ private void loadGroupClass(InstructionGroup group) {
     private void createGroupClassType(InstructionGroup group) {
         String s = classNode.name;
         int lastSlash = classNode.name.lastIndexOf('/');
-        String groupClassInternalName = (lastSlash >= 0 ? s.substring(0, lastSlash) : s)+ '/' + group.getName();
+        String groupClassInternalName = (lastSlash >= 0 ? s.substring(0, lastSlash) + '/': "") + group.getName();
```

Comment:
Just curious: What was the problem here?

Reply:
The test `NoPackageParser` failed because `groupClassInternalName ` started with a slash when using the old logic.
I think that `/SomeClass` is different from `SomeClass`. The first variant is probably interpreted as if `SomeClass` resides in a package with an empty name "" and not the default package.

Reply:
ok, thank you!